Docs: Clarify explanation of RouterInput and RouterOutput types (#4315)
trpc/trpc: www/docs/client/infer-types.md

// @include: server
```

> Using the helpers, you can infer the types of the procedures. The following example shows how to infer the types of the `create` procedure under the `post` namespace, using the example `appRouter`:

Reviewer: Thanks for the suggestion, I agree it could benefit from a little revision, though I'm not sure it's quite there yet.

```suggestion
Using the helpers, we can infer the types of our router. The following example shows how to infer the types of the `post.create` procedure:
```

What do you think of this?

If you agree then please accept this suggestion or enable write access to your PR/fork so I can commit it. That way you retain your contributor credit! 🙂
